### ISSUE TITLE
Safari: Missing visual focus and cannot tab on checkout steps

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#5036]
<img width="1199" alt="Screen Shot 2019-10-16 at 1 54 00 PM" src="https://user-images.githubusercontent.com/45409515/66945101-7689be80-f01c-11e9-8616-26233aeb0118.png">

**Comment by developpeurweb** [comment]
This seems to work on my end now, as per screenshot.

![Screen Shot 2019-11-26 at 4 07 44 PM](https://user-images.githubusercontent.com/15113219/69684490-d5643e00-1086-11ea-8371-8ad44aa99c0f.png)

In order to re-test pls enable the following:
1. Theme Calydon
2. Safari > Preferences > Advanced > Press Tab to highlight each item on a webpage

![Screen Shot 2019-11-26 at 4 08 17 PM](https://user-images.githubusercontent.com/15113219/69684604-3b50c580-1087-11ea-93ef-59fc5a0eb154.png)

**Comment by KateChuen** [comment]
Tested on dev3. It works after enabling in safari (#5041)
